tests/integration/test_parity.py

```diff
@@ -297,6 +297,46 @@ def test_eth_newBlockFilter(self, web3):
         pytest.xfail('Parity returns latest block on first polling for new blocks')
         super().test_eth_newBlockFilter(web3)
 
+    def test_eth_replaceTransaction(self, web3, unlocked_account):
+        pytest.xfail('Needs ability to efficiently control mining')
+        super().test_eth_replaceTransaction(web3, unlocked_account)
+
+    def test_eth_replaceTransaction_already_mined(self, web3, unlocked_account):
+        pytest.xfail('Parity is not setup to auto mine')
```

Comment:
right?

Reply:
Yes, I believe that's correct. cc @dylanjw